meta-lxatac-software/recipes-core/images/lxatac-core-image-base.bb

```diff
@@ -5,12 +5,14 @@ IMAGE_FEATURES = "ssh-server-openssh empty-root-password tools-debug lic-pkgs"
 IMAGE_FSTYPES += "ext4"
 
 # use a fixed directory hash seed to reduce the image delta size
+# nooelint: oelint.vars.specific
```

Comment:
Yes. Actually, these are not really overrides but use the same syntax here.

For `IMAGE_CMD` this is quite clear, but for `EXTRA_IMAGECMD` I am not really sure how this works.